typescript/sdk/src/token/EvmERC20WarpModule.ts

```diff
@@ -52,15 +60,105 @@ export class EvmERC20WarpModule extends HyperlaneModule<
   /**
    * Updates the Warp Route contract with the provided configuration.
    *
-   * @remark Currently only supports updating ISM or hook.
-   *
    * @param expectedConfig - The configuration for the token router to be updated.
    * @returns An array of Ethereum transactions that were executed to update the contract, or an error if the update failed.
    */
   public async update(
-    _expectedConfig: TokenRouterConfig,
+    expectedConfig: TokenRouterConfig,
+  ): Promise<AnnotatedEV5Transaction[]> {
+    TokenRouterConfigSchema.parse(expectedConfig);
+    const actualConfig = await this.read();
+
+    const updateTransactions: AnnotatedEV5Transaction[] = [];
+    updateTransactions.push(
+      ...(await this.updateIsm(actualConfig, expectedConfig)),
+    );
+
+    return updateTransactions;
```

Comment:
return the result of updateIsm directly
```suggestion
    return this.updateIsm(actualConfig, expectedConfig);
```

Reply:
This was a relic of the previous commit when I had the filler logic for hook updates. Updated!